src/commands/org/open.ts

```diff
@@ -16,10 +18,10 @@ import {
 import { Connection, Logger, Messages, Org, SfdcUrl, SfError } from '@salesforce/core';
 import { Duration, Env } from '@salesforce/kit';
 import { MetadataResolver } from '@salesforce/source-deploy-retrieve';
```

Comment:
perf comment: it might be worth doing a dynamic import of this inside the `generateFileUrl` method because there's a decent change it's not needed.